extensions/ql-vscode/src/mocks/mock-gh-api-server.ts

```diff
@@ -0,0 +1,19 @@
+export function startServer(): void {
```

Comment:
I'm fine with merging it like this, but do you think it would make sense to create a class for this instead so we can more easily track state (and potentially test it)? For example, it would be really useful to keep track of whether the server is listening.

Reply:
Yeah might as well do it now (I thought it'll likely be a class but didn't want to do it before we had to)!